### DISCUSSION TITLE
Column size unit

### DISCUSSION THREAD

**Discussion by andreyatx** [KevinVandy/material-react-table#1296] [Q&A]
What measurement unit MRT uses? Docs says it uses pixels size unit. But i set minSize: 61.45, with layoutMode: "grid" to achieve 240px width. Ratio 240/61.45 does not give any clue.

**Comment by KevinVandy** [comment]
depends which layout you are using. In the normal semantic layout, the number will act more as a percentage and let the columns grow. In every case the, the unit is `px`. Read how it works in the docs: https://www.material-react-table.com/docs/guides/column-size
